--- libfyaml-master/src/lib/fy-parse.c	2024-04-22 18:20:09.000000000 +0100
+++ libfyaml-master/src/lib/fy-parse.c	2024-04-23 13:43:15.860783055 +0100
@@ -34,6 +34,7 @@
 #define ATOM_SIZE_CHECK
 #endif
 
+#if 0
 const char *fy_library_version(void)
 {
 #ifndef VERSION
@@ -43,6 +44,7 @@
 	return VERSION;
 #endif
 }
+#endif
 
 int fy_parse_input_append(struct fy_parser *fyp, const struct fy_input_cfg *fyic)
 {
--- libfyaml-master/src/lib/fy-composer.h	2024-04-22 18:20:09.000000000 +0100
+++ libfyaml-master/src/lib/fy-composer.h	2024-04-23 11:27:56.481688585 +0100
@@ -48,7 +48,7 @@
 
 struct fy_composer *fy_composer_create(struct fy_composer_cfg *cfg);
 void fy_composer_destroy(struct fy_composer *fyc);
-int fy_composer_process_event(struct fy_composer *fyc, struct fy_event *fye);
+enum fy_composer_return fy_composer_process_event(struct fy_composer *fyc, struct fy_event *fye);
 
 struct fy_composer_cfg *fy_composer_get_cfg(struct fy_composer *fyc);
 void *fy_composer_get_cfg_userdata(struct fy_composer *fyc);
